priv/repo/migrations/20240929163620_add_classifier_model_name.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Excision.Repo.Migrations.AddClassifierModelName do
+  use Ecto.Migration
+
+  def change do
+    # add nullable column model_name
+    alter table(:classifiers) do
+      add :model_name, :string, null: true
```

Comment:
can we call this base_model_name? and comment that `null: true` means distilbert, and that it is not safe to change the default until we migrate these legacy records' `model_name`s?

Reply:
updated